examples/ui-prompting-examples/src/BuildingBlock.tsx

```diff
@@ -186,7 +186,7 @@ export const BuildingBlockQuestions = (props: {
                             onEvent: (name, event) => {
                                 // ToDo
```

Comment:
Why is the ToDo comment here?

Reply:
it is obsolete, removed -> 3ec1f7ad04a693b1587327a070c434e111bb6a07